src/api/.keystone/config.js

```diff
@@ -60,6 +60,45 @@ var APPLICATION = {
     }
   }
 };
+var PASSWORD = {
+  RANDOM_BYTES_SIZE: 32,
+  STRING_TYPE: "hex",
+  PBKDF2: {
+    ITERATIONS: 1e4,
+    KEY_LENGTH: 64,
+    DIGEST_ALGORITHM: "sha512"
+  }
+};
+var EMAIL_TEMPLATE_IDS = {
+  ACCOUNT: {
+    CONFIRM_EMAIL: "24022e94-171c-4044-b0ee-d22418116575"
```

Comment:
Should we not have centralised EMAIL template IDs please?

Reply:
We have centralised `api/constants.ts` -   this file, `.keystone/config.js` is auto generated by keystone from build script, have no control over this file.